### PR TITLE
De-duplicate platform support and identifier docs

### PR DESCRIPTION
**Pull request by travier**
Avoid duplication (and thus outdated information) and consolidate the
supported platform list in its dedicated page outside of the FAQ.

### CODE REVIEW

**Comment by dustymabe** [comment]
Hmm.. While what we currently have is problematic we (FCOS) don't support every platform that Ignition supports, so the update is misleading IMO. For example we don't have a Brightbox FCOS image we produce.

Similarly we support and document running on things like the Raspberry Pi4. Not sure if we should list things like that here or not.

**Comment by travier** [comment]
We might need to add arch support info too indeed.

File: modules/ROOT/pages/platforms.adoc [hunk 1]
```diff
@@ -1,12 +1,31 @@
-= Platforms
+= Supported platforms
 
 Fedora CoreOS is provisioned via prebuilt disk images, and configured on first-boot via https://github.com/coreos/ignition[Ignition]. Each platform may require specific logic and components, thus dedicated images are provided for each supported environment. Additionally, a unique platform ID is available in the host environment for runtime introspection.
 
-== Runtime introspection
+The currently supported platforms and their identifier are:
```
Comment (dustymabe) [inline modules/ROOT/pages/platforms.adoc:5]: Should this be plural?

```suggestion
The currently supported platforms and their identifiers are:
```

**Review by dustymabe — APPROVED** [review]
LGTM

**Review by dustymabe — APPROVED** [review]
LGTM

**Pull request merged**